docs/architecture/integration-systems.md

```diff
@@ -0,0 +1,56 @@
+# Integration Systems 
+
+Integration System is any system that works in the context of multiple tenants, managing multiple Applications or Runtimes. 
+It exists in a global scope without a tenant assignment.
+Thanks to introducing Integration System, Applications can be completely unaware of Compass, where all 
+integration between Application and Compass is managed by the Integration System.
+
+## GraphQL API Changes
+
+GraphQL API is extended with mutations for managing Integration System, as well as 
+managing its Client Credentials.
+```
+type IntegrationSystem {
+    id: ID!
+    name: String!
```

Comment:
Do we have some special requirements regarding the name? In the ApplicationCreateInput we refer to this name so there may be some rules that limit the naming.